### PR TITLE
Scopes: Format Fixes

### PR DESCRIPTION
**Pull request by Gala-G**
MIPs 106, 107, 108, 113

@patrick-j-govalpha 
@blimpa

### CODE REVIEW

File: MIP106/MIP106.md [hunk 1]
```diff
@@ -34,17 +34,27 @@ The Support Advisory Council is a group of Ecosystem Actors that have been appro
 
 Members of the Advisory Council are directly approved by Maker Governance through a governance poll, and must fulfill specific criteria.
 
-##### 1.1.2.1: The Support Facilitators must ensure that potential Advisory Council Members can apply to be approved by Maker Governance, using an open process with clear instructions.
+##### 1.1.2.1
```
Comment (ghost) [inline MIP106/MIP106.md:37]: Colons `:` after every number please

```suggestion
##### 1.1.2.1:
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by Gala-G** [comment]
@patrick-j-govalpha 
@blimpa

**Comment by blimpa** [comment]
LGTM :)

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**